PlayerControls/sources/ContentControlsUIProps.swift

```diff
@@ -29,6 +29,7 @@ extension DefaultControlsViewController {
         var seekerViewProgress: CGFloat
         var seekerViewBuffered: CGFloat
         var seekerViewAccessibilityLabel: String
+        var seekerViewCuePoints: [Props.Progress.NativeValue]
```

Comment:
Why not just `Props.Progress`?

Reply:
Because then mapping on UIProps apply is required - arrays will not be equal by Type.

Reply:
Ok got it